web/test/pages/onboarding/onboarding-owning.test.tsx

```diff
@@ -82,6 +82,15 @@ describe("onboarding - owning a business", () => {
       expect(screen.getByTestId("banner-alert-REQUIRED_REVIEW_INFO_BELOW")).toBeInTheDocument();
```

Comment:
So the test above this is "prevents user from moving after Step 1 if you have not entered a sector" but this isn't accurate anymore and should only apply to Guest Oscars right? So this test should be deleted I believe?

Reply:
This test definitely should have been failing before. If I remember correctly, seeing that this existed was the reason we wrote this bug.

This might be able to be deleted, but it highlights that we should be distinctly covering for both guest and authenticated users.

Reply:
Okay, so I was thinking that but if you're authenticated don't you already have had to go through onboarding? Like isn't the onboarding only available for guests in the first place? @jphechter

Or is that not the case